cmd/vic-machine/list/list.go

```diff
@@ -114,11 +114,10 @@ func (l *List) prettyPrint(op trace.Operation, cli *cli.Context, vchs []*vm.Virt
 			version = vchConfig.Version.ShortVersion()
 			upgradeStatus = l.upgradeStatusMessage(op, vch, installerVer, vchConfig.Version)
 		}
-
-		parentPath := path.Dir(path.Dir(vch.InventoryPath))
-		name := path.Base(vch.InventoryPath)
+		// When the VCH was found the inventory path was overwritten with the resource pool path, so
+		// to print the path the pool we need to call Dir twice.
```

Comment:
Minor: `print the path to(?) the pool`

Reply:
@cgtexmex Could you be explicit about _which_ pool you're wanting to print and which pool is in the vch.InventoryPath.
If we need to strip two path elements I'd _assume_ that's vch-pool/vch-endpoint which means we're printing the parent pool but that means the InventoryPath should be for the actual endpointVM, not the resource pool mentioned in the comment.